config/ASU_user.mk

```diff
@@ -0,0 +1,32 @@
+####################
+CXX           ?= g++
+NVCC          ?= nvcc
+LINKER        ?= g++
+CPPFLAGS      ?=
+CXXFLAGS      ?= -O3 -Wall -pedantic
+NVCCFLAGS     ?= -O3 -Xcompiler "-Wall" -lineinfo
+LDFLAGS       ?=
+DOXYGEN       ?= doxygen
+PYBUILDFLAGS   ?=
+PYINSTALLFLAGS ?=
+
+GPU_ARCHS     ?= 50 # GeForce GTX TITAN X
+# Always check the GPU Compute Capability (cc) and compatibility with nvcc version.
+
+# Install may work but the usgae code throws the following error for mismatched GPU and
+# cudaGetErrorString(cuda_ret) = no kernel image is available for execution on the device
+#  or
+# cudaGetErrorString(cuda_ret) = invalid device function
```

Comment:
I'm not sure I understand these comments. "for mismatched GPU and..."? What is the user to do if they get these errors?

Reply:
The solution would be to change the `GPU_ARCHS` variable in `user.mk` to the correct sm version and re-compile.

Reply:
So is this relevant here? I mean, this file is specifically for an ASU (read: intrepid) install, so we already know the GPU.

Reply:
If it is specifically for ASU then, no, I don't think you need it.  It might be nice to put in what the correct values are, though.